.storybook/preview.js

```diff
@@ -56,51 +55,18 @@ const SORT_ORDER = {
 };
 
 export const parameters = {
-  layout: 'centered',
+  viewMode: 'docs',
+  previewTabs: { 'storybook/docs/panel': { index: -1 } },
   actions: { argTypesRegex: '^on.*' },
   options: {
     storySort: sortStories(SORT_ORDER),
   },
   docs: { theme, components },
 };
 
-const StoryStyles = styled.div`
-  display: flex;
-  flex-direction: column;
-  gap: 1rem;
-  justify-content: center;
-  align-items: center;
-  min-height: 100vh;
-  .sbdocs & {
-    min-height: auto;
-  }
-`;
-
-const globalStyles = (theme) => css`
-  code {
-    padding: 0.15em 0.2em 0.05em;
-    margin: -0.15em 0 -0.05em;
-    border-radius: 0.2em;
-    background: ${theme.colors.n200};
-  }
-
-  pre code {
-    background: inherit;
-  }
-`;
-
-const withStoryStyles = (Story) => {
-  return (
-    <StoryStyles>
-      <Story />
-    </StoryStyles>
-  );
-};
-
 const withThemeProvider = (Story) => (
   <ThemeProvider theme={light}>
     <BaseStyles />
-    <Global styles={globalStyles} />
```

Comment:
This was apparently being overridden, our code block styles are coming from Storybook (I guess). I think they look fine

Reply:
This must have broken recently. I'm not happy with how inline code looks:

![Screenshot 2022-01-05 at 17 54 01](https://user-images.githubusercontent.com/11017722/148256948-1aab03ac-5d9e-4b63-82ee-a50fa0cd297c.png)

This is what it used to (and IMO should) look like: 

![Screenshot 2022-01-05 at 17 56 02](https://user-images.githubusercontent.com/11017722/148257298-a81373ab-cf77-480c-b131-7652d3f631ed.png)

I wonder if there's a different/new way to add styles to the Docs tab 🤔

Reply:
True, it does look clearer with the gray background. I think it's been like this for a while though, because thinking back, I remember thinking that the `code` styles were not very noticeable

Reply:
Using emotion's `Global` component definitely doesn't work (I can't see the styles injected anywhere).

I also noticed that the base styles seem to be injected once for each story (!), which means that the rules are duplicated dozens of times on each page.

I'll handle both of these things since they're related in a separate PR, and merge this one now since those links are still broken on the production SB 🙂

Reply:
Weird, because the BaseStyles use the Global component under the hood 🤔

Yup, I couldn't find a way to inject the BaseStyles only once, so they're included for every story alongside the ThemeProvider [here](https://github.com/sumup-oss/circuit-ui/blob/main/.storybook/preview.js#L70).